_aggregations/metric/cardinality.md

```diff
@@ -94,4 +143,10 @@ GET opensearch_dashboards_sample_data_ecommerce/_search
   }
 }
 ```  
-{% include copy-curl.html %}
\ No newline at end of file
+{% include copy-curl.html %}
+
+## Missing values
```

Comment:
Please confirm accuracy and relevance of this section.

Reply:
Heh, yeah I guess you could supply one of the existing values. I think this is of academic interest anyway. If you're using aggregation to estimate cardinality, I don't think you're going to be too concerned with +/-1 value.